fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractCompartmentTest.java

```diff
@@ -39,54 +38,68 @@
     Practitioner savedPractitioner;
     RelatedPerson savedRelatedPerson;
     Observation savedObservation;
-    
+    Observation savedObservation2;
+
     /**
      * Builds and saves an Observation with the following references:
-     * 
+     *
      * Observation.subject = Patient
      * Observation.device = Device
      * Observation.encounter = Encounter
      * Observation.performer[0] = Patient
      * Observation.performer[1] = Practitioner
      * Observation.performer[2] = RelatedPerson
+     *
+     * Builds and saves an Observation with the following logical ID-only reference:
+     *
+     * Observation.subject = Patient
      */
     @BeforeClass
     public void createResources() throws Exception {
         Observation.Builder observationBuilder = ((Observation) TestUtil.readExampleResource("json/ibm/minimal/Observation-1.json")).toBuilder();
-        
+        Observation.Builder observation2Builder = ((Observation) TestUtil.readExampleResource("json/ibm/minimal/Observation-1.json")).toBuilder();
+
         Patient patient = TestUtil.readExampleResource("json/ibm/minimal/Patient-1.json");
         savedPatient = persistence.create(getDefaultPersistenceContext(), patient).getResource();
         observationBuilder.subject(buildReference(savedPatient));
         observationBuilder.performer(buildReference(savedPatient));
-        
+        observation2Builder.subject(Reference.builder().reference(string(savedPatient.getId())).build());
```

Comment:
Does the prior line buildReference accomplish the same as the code on 66? might be best to reuse if it does the same thing.

Reply:
we seem to bounce between static import and this method, maybe something to reconcile next time we're in the code. 

Treat this as a thought for the future.

Reply:
The buildReference() method is not the same - it builds a valid relative reference containing both resource type and logical ID. On line 66, I want just the logical ID.